server/src/main/scala/org/http4s/server/staticcontent/WebjarService.scala

```diff
@@ -52,7 +52,7 @@ object WebjarService {
     * @param config The configuration for this service
     * @return The HttpService
     */
-  def apply[F[_]: Monad: Sync](config: Config[F]): HttpService[F] = Service {
+  def apply[F[_]: Monad: Sync](config: Config[F]): HttpService[F] = HttpService.lift {
```

Comment:
I'm going to advocate for deprecating `HttpService.lift` in #1424.

Reply:
I support that. So we will have `apply(pf: PartialFunction[Request[F], F[Response[F]]])` and `liftF(f: Request[F] => OptionT[F, Response[F]])` as the ways to create an `HttpService[F]`, right?